### PR TITLE
Navbar adjustment

### PR DESCRIPTION
**Pull request by tomquinonero**
Small adjustments for theming :)

### CODE REVIEW

**Review by carsoli — APPROVED** [review]

**Review by FlorianWendelborn — APPROVED** [review]

**Comment by carsoli** [comment]
@tomquinonero we both approve but please don't merge if you will need to publish this ASAP

we would rather publish from a separate branch cause master has merged changes that are un-publishable

**Pull request merged**
